src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -269,5 +296,18 @@ static ToolWindowPane ShowToolWindow(Guid windowGuid)
                 return sp.TryGetService(serviceType);
             }
         }
+
+        IVSGitExt CreateVSGitExt(IGitHubServiceProvider sp)
+        {
+            switch (VSVersion.Major)
```

Comment:
In the past I've always used `DTE.Version` to differentiate versions of Visual Studio. This always returns a string with the major version number with a `.0` (even for different minor versions). I used `VSVersion` because it was there, but I think it might hit the file system when reading the file version number.